src/widgets/Ruler.cpp

```diff
@@ -702,18 +702,3 @@ void Ruler::SetCustomMinorLabels(
    }
 }
 #endif
-
-void Updater::Label::Draw(wxDC&dc, bool twoTone, wxColour c) const
```

Comment:
The fifth commit moves this function.

I'd rather just squash that into the commit that introduces class `Updater`.  It's not recommended to have the non-inline definitions of a class's member functions in more than one file: so, avoid putting things into this state to begin with.